payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/RowElementUI.kt

```diff
@@ -20,6 +20,13 @@ internal fun RowElementUI(
     val fields = controller.fields
     val cardStyle = CardStyle(isSystemInDarkTheme())
 
+    var numVisibleFields = fields.size
+    fields.forEach {
```

Comment:
A more functional way would be 
`var numVisibleFields = fields.filter { !hiddenIdentifiers.contains(it.identifier) }.size`